versioned_docs/version-8.4/reference/supported-environments.md

```diff
@@ -38,6 +38,7 @@ For example, 1.2+ means support for the minor version 2, and any higher minors (
 - **Zeebe Java Client**: OpenJDK 8+
 - **Zeebe Go Client**: Go 1.13+
 - **zbctl**: Windows, MacOS, and Linux (latest)
+- **Connectors SDK**: OpenJDK 21+
```

Comment:
@chillleader I think we did not backport the SDK compiler settings. Hence its 21 for 8.4